lib/twinklyhaha/application.ex

```diff
@@ -8,7 +8,7 @@ defmodule Twinklyhaha.Application do
   def start(_type, _args) do
     children = [
       # Start the Ecto repository
-      # Twinklyhaha.Repo,
+       Twinklyhaha.Repo,
```

Comment:
Since we are not using the database for now, feel free to remove repo from the application children